src/components/enterprise/validate.js

```diff
@@ -0,0 +1,21 @@
+import { isEmail } from "validator";
+
+const validate = values => {
```

Comment:
Why aren't we using Yup or Joi? There's no meaningful validation here. We could be properly validating email addresses, and urls at a mimimum (could also validate phone numbers based on length)

Reply:
We could also use Yup to validate ABN numbers based on length (strip all whitespace before validating though, that way the user can add spaces)

Reply:
As discussed, keep `validator`, add validation for urls, (validate abn on length, same for phone?)